src/rewriter.fs

```diff
@@ -267,12 +264,25 @@ let reorderTopLevel t =
     else
         t
 
+let rec iterateSimplifyAndInline li =
+    let foundInlinable =
+        if options.noInlining then
+            false
+        else
+            let foundInlinableRef = ref false
+            let mapExpr _ = function e -> e
```

Comment:
nit. can be written `let mapExpr _ e = e`

Reply:
:facepalm: